perun-core/src/main/java/cz/metacentrum/perun/core/bl/ServicesManagerBl.java

```diff
@@ -254,11 +254,12 @@
 	 *
 	 * @param perunSession
 	 * @param service
+	 * @param forceFlag if set to true, removes the service with all dependendant objects from db instead of raising exception
 	 * @throws RelationExistsException
 	 * @throws InternalErrorException
 	 * @throws ServiceAlreadyRemovedException if there are 0 rows affected by deleting from DB
 	 */
-	void deleteService(PerunSession perunSession, Service service) throws RelationExistsException, ServiceAlreadyRemovedException;
+	void deleteService(PerunSession sess, Service service, boolean forceFlag) throws RelationExistsException, ServiceAlreadyRemovedException;
```

Comment:
There was no reason to change the parameter name perunSession to sess. If you really want to change it, you need to change also Javadoc properly.

Reply:
Renamed.